### PR TITLE
return overall_score from MTBenchBranch.generate_judgement()

### PR DESCRIPTION
**Pull request by alimaredia**
This allows the overall_score to be shown by
callers of the library along with qa pairs
and the error rate.

### CODE REVIEW

**Comment by danmcp** [comment]
@sallyom FYI on the api change

**Comment by danmcp** [comment]
Note: This can't merge until a change is made to the cli to allow for the 2 and 3 param tuple return at the same time.

**Review by danmcp — APPROVED** [review]
Looks good, just need to fix the formatting error

**Review by nathan-weinberg — APPROVED** [review]

**Review by alinaryan — APPROVED** [review]

File: src/instructlab/eval/mt_bench.py [hunk 1]
```diff
@@ -260,10 +260,12 @@ def judge_answers(
             serving_gpus    Number of gpus allocated for serving.  Used to tune with max_workers=auto.  None indicates to use value specified in constructor.
 
         Returns:
+            overall_score   overall score from the evaluation
             qa_pairs        Question and answer pairs (with scores) from the evaluation
+            error_rate      percentage of questions dropped due to errors during evaluation
```
Comment (danmcp) [inline src/instructlab/eval/mt_bench.py:265]: If you do make another change to this commit, this is also missing for the mt_bench case above

**Pull request merged**